starter/tslint.json

```diff
@@ -2,9 +2,9 @@
   "extends": [
     "tslint:latest",
     "tslint-sonarts",
-    "codelyzer",
     "tslint-config-prettier",
```

Comment:
This `tslint-config-prettier` should be at the end because it will override any conflicting rule that is managed by Prettier ;)